appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: 1.9.0.{build}
+environment:
+  TOKEN:
+    secure: YOUR-ENCRYPTED-CHOCO-APIKEY

```

Comment:
can you take a look at my comment on the PR? I wonder what the publishing lifecycle looks like.